addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -5480,8 +5481,11 @@ void StartMusic(int client=0)
 		for(int target; target<=MaxClients; target++)
 		{
 			playBGM[target] = true;  //This includes the 0th index
+			if(target>0)	//GetClientUserId(target) will not work with target = 0

```

Comment:
You may need a IsClientInGame / IsValidClient check.

Reply:
Effectively there must be an _IsValidClient()_. Also, as _IsValidClient_ validate already the `client>0`, it just should be like:
```suggestion
			if(IsValidClient(target))

```